File: Der Satz des Heron/Satz_des_Heron.py
import math

def get_area():
    a = 3
    b = 4
    c = 5
    s = ( a + b + c ) / 2
    return math.sqrt( s * ( s - a) * ( s - b ) * ( s - c ) )

area = get_area()
print(area)

File: Der Satz des Heron/Satz_des_Heron.js

let a = 3;
let b = 4;
let c = 5;

function get_area(a, b, c) {

    let s = ( a + b + c ) / 2;

    return Math.sqrt( s * ( s - a ) * ( s - b ) * ( s - c ) );
}

let area = get_area(a, b, c)

console.log(area);

